src/bench.c

```diff
@@ -799,9 +799,21 @@ int benchmark_all(void)
 		if ((format->params.flags & FMT_DYNAMIC) ||
 		    strstr(format->params.label, "-opencl") ||
 		    strstr(format->params.label, "-ztex") ||
-		    !strcmp(format->params.label, "crypt"))
+		    !strcmp(format->params.label, "crypt")) {
+#if defined(HAVE_OPENCL)
+			if (strstr(format->params.label, "-opencl")) {
+				/* Allow OpenCL build's "--test" to run on no-OpenCL systems */
+				/* It is a like a hack, but it is necessary since OpenCL must be started */
+				/* to get the number of devices. The init[1] itself is performed only once. */
+				/* [1] Enumeration of platforms and devices; and parse options. */
```

Comment:
I'd use a single multi-line comment like:

```
/*
 * Allow OpenCL build's "--test" to run on no-OpenCL systems.
 * It is a hack, but it is necessary since OpenCL must be started
 * to get the number of devices. OpenCL initialization (enumeration
 * of platforms and devices, option parsing) is performed only once.
 */
```

And yes, no indentation of the comment to the code block's indentation depth.